api/audit/models.py

```diff
@@ -72,15 +76,15 @@ def get_history_record_model_class(
         module = import_module(module_path)
         return getattr(module, class_name)
 
+    @property
+    def can_related_object_type_change_feature_value_for_sdk(self) -> bool:
+        # Returns true if the audit log have an object type that can change the
+        # feature value for the sdk endpoints
+        return self.related_object_type != RelatedObjectType.CHANGE_REQUEST.name
+
     @hook(AFTER_SAVE)
     def update_environments_updated_at(self):
```

Comment:
Since we're now actually triggering the SSE messages, the name of this method should probably change?